maintenance_github_grooming.es.Rmd

```diff
@@ -1,39 +1,39 @@
-# GitHub Grooming {#grooming}
+# Gestión de GitHub {#grooming}
 
 ```{block, type='summaryblock'}
-Los paquetes de rOpenSci se desarrollan actualmente en su gran mayoría en GitHub.
-Aquí tienes algunos consejos para aprovechar la plataforma en una sección sobre [hacer que tu repo sea más descubrible](#repodiscoverability) y una sección sobre [publicitar tu propia cuenta de GitHub después de pasar por la revisión por pares](#marketown).
+Actualmente la gran mayoría de los paquetes de rOpenSci se desarrollan en GitHub.
+Aquí tienes algunos consejos para aprovechar la plataforma en una sección sobre [hacer que tu repo sea más fácil de descubrir](#repodiscoverability) y una sección sobre [publicitar tu propia cuenta de GitHub después de pasar por la revisión por pares](#marketown).
 ```
 
 ## Haz que tu repositorio sea fácil de descubrir {#repodiscoverability}
 
-### Temas de repositorio en GitHub
+### Áreas de repositorio en GitHub
 
-Los [temas del repositorio](https://blog.github.com/2017-01-31-introducing-topics/) de GitHub ayudan a navegar y buscar en los repositorios de GitHub, y son usados por [`codemetar`](https://github.com/ropensci/codemetar) en las palabras clave del registro de rOpenSci.
+Las [áreas del repositorio](https://blog.github.com/2017-01-31-introducing-topics/) de GitHub ayudan a navegar y buscar en los repositorios de GitHub, y son usados por [`codemetar`](https://github.com/ropensci/codemetar) en las palabras clave del registro de rOpenSci.
```

Comment:
Why the change of the wodk from temas to áreas?

Reply:
Me pareció que se entendía mejor. "Tema" me suena a "tema la vaca"; o sea, "de qué se trata". Me pareció que con "área" se entendía mejor que es más un "a qué área de estudio pertenece". Quizás se puede usar "áreas temáticas" o algo más. O dejar "tema" :p